examples/networking-gre-playbook/site.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Configure GRE Tunnel between rtr1 and rtr2
+  hosts: routers
+  gather_facts: no
+  connection: local
+  vars:
+     rtr1_public_ip: "34.236.147.137"
+     rtr2_public_ip: "54.209.50.0"
+  tasks:
+  - name: create tunnel interface to R2
+    ios_config:
+      lines:
+       - 'ip address 10.0.0.1 255.255.255.0'
+       - 'tunnel source GigabitEthernet1'
+       - 'tunnel destination {{rtr2_public_ip}}'
+      parents: interface Tunnel 0
+    when:
+      - '"rtr1" in inventory_hostname'
```

Comment:
Really? Hardcoding hostnames is common practice int he networking space? Coming form the linux sysadmin space this is an anti-pattern in Ansible.

Reply:
The problem here is to address one host of a fixed set of two hosts. Similar situation as you have it with HA systems: you have to address first one host and then the other somehow. I'm open for better ideas.

Reply:
Yes. I think we need to work on this one. @gundalow do you have any additional thoughts? Will think about it myself.